__mocks__/schemas/PlaygroundSourceSchema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "PlaygroundSourceSchema.xsd",
+  "type": "XML",
+  "targetNamespace": "http://tempuri.org/Source.xsd",
+  "namespaces": { "ns0": "http://tempuri.org/Source.xsd", "xs": "http://www.w3.org/2001/XMLSchema" },
+  "schemaTreeRoot": {
+    "key": "/ns0:PlaygroundRoot",
+    "name": "PlaygroundRoot",
```

Comment:
Same comment as above to add in Source/Target to the path so it's easy to see at a glance